features/ui/input/input.tsx

```diff
@@ -97,7 +101,10 @@ export const Input = ({
         error={error}
         errorMsg={errorMsg}
         value={value}
-        onChange={(e) => setValue(e.target.value)}
+        onChange={(e) => {
+          setValue(e.target.value);
```

Comment:
I'd probably let the parent be responsible for managing the `value` state here. So basically remove the state from this component and accept a `value` prop. This way the parent can "control" the input component

Reply:
Yes, this would make more sense as I find it quite difficult to really 'control' the input component when using the component.